### ISSUE TITLE
Revision not found when diffing history

### ISSUE THREAD

**Issue by michaelWuensch** [ProjectBorealis/UE4GitPlugin#187]
I want to review code from other team members. It seems that this plugin has problems diffing certain commits.
As far as I can see it does not work when a version of an LFS file was never fetched.

The following example outlines the history of File A (from Bottom to top)
Commit 3: File A - changed by teammate  <- locally available
Commit 2: File A - changed by teammate   <- not locally available, lfs file was not fetched
Commit 1: File A - changed by me   <- locally available

When I am locally on commit 1 and I then pull the newest changes from remote which includes multiple revisions of the SAME file, not all revisions of that file will be fetched, only the most recent one.

Now when going to the Asset with right click -> RevisionControl -> History
I can compare Commit 3 with Commit 1, but not Commit 3 with Commit 2.

Can anything be done about it, like requesting to fetch the neede revision instead of showing
"Unable to load assets to diff. Content may no longer be supported?"

**Comment by mastercoms** [comment]
Hi, are you using the dev branch or release?

**Comment by michaelWuensch** [comment]
I tested with Release 3.16.
The official Unreal git integration has the same problem.

**Comment by mastercoms** [comment]
Yes, a fix was made for this, I'm not sure why it's not working for you.
